doc/sphinx/src/using-eos.rst

```diff
@@ -445,6 +711,23 @@ unmodified EOS model, call
 
 The return value here will be either the type of the ``EOS`` variant
 type or the unmodified model (for example ``IdealGas``) or, depending
+
```

Comment:
What going on here @Yurlungur ?

Reply:
great question. Looks like I accidentally introduced a ton of whitespace. Gonna blame my cat and just remove it lol.